packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.ts

```diff
@@ -20,10 +20,12 @@ import {
   type OnUnknownPluginMessage,
   type OnPromptAIMessage,
   type PromptAIPayload,
+  OnPreviewDimensionMessage,
```

Comment:
If this is a type, then let's explicitly say it here:
```suggestion
  type OnPreviewDimensionMessage,
```